climate.py

```diff
@@ -31,14 +32,23 @@
 }
 FAN_MODES = {
     'LOW': c_const.FAN_LOW,
-    'LOW_MID': 'low-mid',
     'MID': c_const.FAN_MEDIUM,
-    'MID_HIGH': 'mid-high',
     'HIGH': c_const.FAN_HIGH,
 
-    'NATURE': 'nature',
-    'POWER': 'power',
+    'NATURE': 'auto',
+    'POWER': 'jet',
 }
```

Comment:
It seems like this change is unrelated, right?

Reply:
true, i was adjusting a bit locally. shouldn't have reached the pr